dist/tingle.css

```diff
@@ -6,33 +6,30 @@
   box-sizing: border-box;
 }
 
+.tingle-modal *:focus {
+  outline: 1px solid black;
```

Comment:
I would prefer to keep the browser's default focus ring instead of a 1px black ring. It's not very noticeable, especially around the close button.

Maybe we could make a nicer focus ring with box-shadow, but it's a bit unsafe knowing that the focusable elements in the modal's content may have box-shadow already.

I would like to have the @robinparisi opinion on that.